ci/aws-analytical-env/shared/meta.yml

```diff
@@ -214,3 +214,47 @@ meta:
       params:
         TF_CLI_ARGS_plan: -lock-timeout=300s
         TF_INPUT: "false"
+
+    mirror-git-repo:
+      task: mirror-git-repo
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        params:
+          AWS_DEFAULT_REGION: ((dataworks.aws_region))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
```

Comment:
not sure what the intention was here, but I don't think this does anything

Reply:
AWS_DEFAULT_REGION is required , If I removed its fails with `Fatal error: Failed to connect to proxy URL:`

Reply:
yes, but that is picked up from the env var set on line 229 isn't it?

Reply:
Yes , Dan Amended code